spec/index.html

```diff
@@ -66,7 +66,7 @@
     doJsonLd:             true,
 
     // name (with the @w3c.org) of the public mailing to which comments are due
-    wgPublicList: "public-rch-wg@w3.org"
+    wgPublicList: "public-rch-wg"
```

Comment:
Not that it matters much at the moment, but I'm thoroughly confused by these lines.

The comment-instruction *appears* to say that `public-rch-wg@w3.org` is the desired value (being `with the @w3c.org`), but the change just merged is to `public-rch-wg`.

If this is the intended meaning of the instruction, I suggest logging a PR against their source, to change `with` to `minus`, and optimally to add an example.

Reply:
I'm not sure the source of that original comment, but the [wiki entry](https://github.com/w3c/respec/wiki/wgPublicList) doesn't use the `@w3.org`. I'll remove the comment.